test/Libraries/DynamoPythonTests/CodeCompletionTests.cs

```diff
@@ -42,8 +42,7 @@ public void SharedCoreCanReturnCLRCompletionData()
 
             Assert.IsTrue(completionList.Any());
             Assert.IsTrue(completionList.Intersect(new[] { "Hashtable", "Queue", "Stack" }).Count() == 3);
-            // Serial tests load an extra type (System.Collections.Immutable) in the Python engine
-            Assert.IsTrue(completionData.Length >= 29 && completionData.Length <= 30);
+            Assert.IsTrue(completionData.Length == 31);
```

Comment:
?

Reply:
there are more types which are imported in net6 vs net48 - I got rid of the zany logic because AFAIK the same number of types seem to be imported when running one or multiple tests.